### PR TITLE
update num thermistors to include all channels

### PR DESCRIPTION
**Pull request by tanvib00**
## 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] I have read and followed the `CONTRIBUTING.md` file
- [ ] I have applied the .clang-format file to my changes
- [ ] I have commented my code using Doxygen style comments (see Contributing MD for examples), particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the design documentation 
- [ ] I have updated any relevant parameters in the internal ICD
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If this is a bug-fix, I have added tests that can catch similar bugs in the future
- [ ] New and existing unit tests pass locally with my changes
- [ ] For cfe apps I have reserved an empty address for my msgs_ids and perf_ids from the [tracking list](https://docs.google.com/spreadsheets/d/1yRUhNTRBOiGLswlsWftNaYlwGKDeNQ_fCnDYFrc_7fU/edit#gid=772812381) and checked that it does not clash with other apps
- [ ] For new added message types, there is a timestamp included via either an explicit timestamp variable in the struct or buried in the a tlmHeader.

### CODE REVIEW

File: fsw/public_inc/heater_control_msgs.h [hunk 1]
```diff
@@ -20,7 +20,7 @@
 #include "moonranger_common_types.h"
 
 #define NUM_HEATERS 14
-#define NUM_THERMISTORS 13
+#define NUM_THERMISTORS 16
```
Comment (tenzincrouch) [inline fsw/public_inc/heater_control_msgs.h:23]: @jtingyu7 and @anju777 - This change will change the size for a couple of payloads. Have you been using the predefined lengths and/or sizeof? I just want to make sure this change doesn't propogate issues elsewhere.

Comment (jtingyu7) [reply fsw/public_inc/heater_control_msgs.h:23]: It does not matter on my side since the obc uses sizeof operator which should self-adjust accordingly

Comment (anju777) [reply fsw/public_inc/heater_control_msgs.h:23]: The only hard coded value I have is the max payload length, which is motor telemetry. As long as the max message length stays the same, nothing should change on my end.

**Review by tenzincrouch — APPROVED** [review]
Approved based on comments above.

**Pull request merged**